packages/components/src/components/Header/Header.js

```diff
@@ -28,14 +27,20 @@ function Header({
   logoutButton,
   onHeaderMenuButtonClick
 }) {
+  function skipToContentClick(event) {
+    event.preventDefault();
+    window.scrollTo(0, 0);
+    document.getElementById('main-content').focus();
+  };
+
   return (
     <CarbonHeader aria-label="Tekton Dashboard" className="tkn--header">
-      <SkipToContent>
+      <a href="#" className="bx--skip-to-content" onClick={skipToContentClick}>
```

Comment:
Can we continue to use the `SkipToContent` component here so we don't have to rely on knowing the classname etc.? Looks like it allow overriding the `href` and adding additional props such as the `onClick`: https://github.com/carbon-design-system/carbon/blob/e7ef150ba1d74e2e864a9d6d67e612e824dcbf51/packages/react/src/components/UIShell/SkipToContent.js#L18